util/data_parser.py

```diff
@@ -249,6 +249,13 @@ def set_up_database(cursor):
         cursor.execute("create table if not exists words(id int(11) not null, word varchar(55), primary key(id))")
 
     @staticmethod
+    def drop_database(cursor):
+        db = DataParser.get_connection()
+        cursor = db.cursor()
+        config = DataParser.get_config()
+        cursor.execute("drop database if exists %s;" % config['database']['database_name'])
+
```

Comment:
This method won't be used in the process method, so call clean_up after you are done executing.